index.css

```diff
@@ -202,32 +234,333 @@ footer {
   column-gap: 40px;
 }
 
+.title {
+  margin-top: 5px;
+  margin-bottom: 5px;
+}
+
 #tabs {
   display: flex;
-  gap: 15px;
+  gap: 10px;
   padding-bottom: 20px;
 }
 
-.lang {
-  font-weight: 550;
-  color: rgba(0, 0, 0, 1);
-  font-size: 22px;
-  display: flex;
-  justify-content: space-between;
-  padding-bottom: 45px;
+.ttwo {
+  padding-left: 15px;
 }
 
 .about {
   font-size: 2.2rem;
   color: rgba(23, 43, 77, 1);
 }
 
-.languages {
-  padding-left: 10px;
-  flex-direction: row;
+.child {
+  width: 303px;
+  height: 66.71px;
+  border-radius: 8px;
+  background-color: rgba(247, 247, 249, 1);
+  margin-bottom: 12px;
   display: flex;
-  gap: 30px;
-  font-weight: 550;
-  padding-bottom: 15px;
+  align-items: center;
+  gap: 20px;
   font-size: 15px;
+  font-weight: 500;
+}
+
+.line {
+  margin-left: -10px;
+  width: 93%;
+  border: none;
+  border-top: 1px solid #ccc;
+}
+
+#LI {
+  margin-bottom: 200px;
+}
+
+#contact-me {
+  font-size: 35px;
+  text-align: center;
+  color: aliceblue;
+  padding-top: 135px;
+}
+
+.contact-me-p {
+  text-align: center;
+  margin: 45px;
+  color: aliceblue;
+}
+
+#form {
+  display: flex;
+  padding: 10px;
+  margin: 4%;
+  margin-bottom: 0;
+  gap: 21px;
+  flex-direction: column;
+  align-items: flex-start;
+  padding-bottom: 30px;
+}
+
+#name {
+  width: 100%;
+  height: 48px;
+  border-radius: 8px;
+  border-style: none;
+}
+
+#email {
+  width: 100%;
+  height: 48px;
+  border-radius: 8px;
+  border-style: none;
+}
+
+#message {
+  width: 99%;
+  height: 170px;
+  border-radius: 8px;
+  border-style: none;
+  font-family: 'Poppins', sans-serif;
+  padding-left: 6px;
+}
+
+.end {
+  align-items: center;
+}
+
+input::placeholder {
+  font-family: 'Poppins', sans-serif;
+  padding-left: 8px;
+}
+
+.myself {
+  background-color: rgba(255, 255, 255, 1);
+  border-top-right-radius: 85px;
+}
+
+#chirvon {
+  padding-left: 25%;
+  padding-right: 10%;
+}
+
+.lang {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 20px;
+  font-weight: 500;
+  line-height: 24px;
+}
+
+.parent {
+  margin-bottom: 30px;
+}
+
+@media (min-width: 750px) {
```

Comment:
@Sohail-Aslam Good job so far but i would suggest you to correct your media query break point, and make it similar to the given instructions 🦖 

`Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: >=768px).`

![sohail-styler](https://github.com/Sohail-Aslam/Mobile-Version/assets/122353030/da6358be-602b-4bbf-bd4a-5407777d8fe4)